graphql/types/paging.graphql

```diff
@@ -0,0 +1,8 @@
+type Paging {
+  page: Int!
+  perPage: Int!
+  total: Int!
+  pages: Int!

```

Comment:
No need for this information because it can be derived. total / perPage

Reply:
yes i agree, do we need to keep minimum set of information here? 
it will be easy for client side development if we calculate those values in graphql side.

Reply:
It's a good practice to keep our API as short as possible. For reference: https://2014.jsconf.eu/speakers/sebastian-markbage-minimal-api-surface-area-learning-patterns-instead-of-frameworks.html

Reply:
Also on your PRs please add a running example of your queries. like:

https://github.com/vtex-apps/store-graphql/pull/370